src/components/AccountModule/ClientSyncedInfo.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { GU } from '@aragon/ui'
+import { useClientBlockNumber } from './useClientBlockNumber'
+import { getClientSyncState } from './utils'
+
+function ClientSyncedInfo({ listening, online, syncDelay }) {
+  const latestClientBlockNumber = useClientBlockNumber()
+  const { header, info } = getClientSyncState(
```

Comment:
Perhaps we can pick better names for these, since it's not really a header anymore (they're both on the same line).

Maybe `state` and `description`?